### PR TITLE
replace wallet with w (moving to w.akroma.io)

### PR DESCRIPTION
**Pull request by detroitpro**
This PR changes the URL for wallet.akroma.io to be w.akroma.io

We are moving the wallet to be hosted by GitHub pages. GitHub provides static content hosting, with SSL.

### CODE REVIEW

File: chrome-extension/signmsg.html [hunk 1]
```diff
@@ -162,8 +162,8 @@ <h5 translate="MSG_signature">
...
+"logo":"https://myetherwallet.com/images/myetherwallet-logo-banner.png","description": "MyEtherWallet.com is a free, open-source, client-side interface for generating Ethereum wallets &amp; more. Interact with the Ethereum blockchain easily &amp; securely. Double-check the URL ( myetherwallet.com ) before unlocking your wallet.","sameAs":["https://w.akroma.io/","https://chrome.google.com/webstore/detail/myetherwallet-cx/nlbmnnijcnlegkjjpcfjclmcfggfefdm","https://www.facebook.com/MyEtherWallet/","https://twitter.com/myetherwallet","https://medium.com/@myetherwallet_96408","https://myetherwallet.github.io/knowledge-base/","https://github.com/kvhnuke/etherwallet","https://github.com/MyEtherWallet","https://kvhnuke.github.io/etherwallet/","https://github.com/kvhnuke/etherwallet/releases/latest","https://github.com/409H/EtherAddressLookup","https://myetherwallet.slack.com/","https://myetherwallet.herokuapp.com/","https://www.reddit.com/r/MyEtherWallet/","https://www.reddit.com/user/insomniasexx/","https://www.reddit.com/user/kvhnuke/","https://www.reddit.com/user/myetherwallet"]}</script>
```
Comment (mkrufky) [inline chrome-extension/signmsg.html]: We might have missed a few instances of "MyEtherWallet.com" conversions to "wallet.akroma.io" into "w.akroma.io", here.

Comment (mkrufky) [reply chrome-extension/signmsg.html]: also facebook twitter medium github.....

Comment (ciripel) [reply chrome-extension/signmsg.html]: remember to change `DON'T GET PHISHED, BOOKMARK https://wallet.akroma.io` to `DON'T GET PHISHED, BOOKMARK https://w.akroma.io`. and also README.md file of this repo

Comment (ciripel) [reply chrome-extension/signmsg.html]: we may want also to change this part and explorer link in https://w.akroma.io/#view-wallet-info
w akroma wallet

Comment (pavelhr) [reply chrome-extension/signmsg.html]: And the link in Awesome Akroma.

**Comment by detroitpro** [comment]
merging and deploying to w.akroma.io 
leaving wallet.akroma.io up
all new deployments will be to w.akroma.io

finding and updating mew refs will be an ongoing process.

**Pull request merged**